server/config/config.go

```diff
@@ -33,6 +33,8 @@ type MysqlConfig struct {
 	MaxOpenConns    int    `yaml:"max_open_conns"`
 	MaxIdleConns    int    `yaml:"max_idle_conns"`
 	ConnMaxLifetime int    `yaml:"conn_max_lifetime"`
+	IAMAccess       bool
+	Region          string
```

Comment:
We don't need these in the yaml?

Reply:
Which ones? The intention is for all configs to be available in yaml.